packages/jackett/build.sh

```diff
@@ -0,0 +1,32 @@
+TERMUX_PKG_HOMEPAGE="https://github.com/jackett/jackett"
+TERMUX_PKG_DESCRIPTION="API Support for your favorite torrent trackers"
+TERMUX_PKG_LICENSE="GPL-3.0"
+TERMUX_PKG_MAINTAINER="@termux"
+TERMUX_PKG_VERSION="0.22.1411"
+TERMUX_PKG_SRCURL="https://github.com/Jackett/Jackett/archive/refs/tags/v${TERMUX_PKG_VERSION}.tar.gz"
+TERMUX_PKG_SHA256="77d7912835de603708188ffa1fcb9006f1304efc50c286dd0c5d0ac1ffe80184"
+TERMUX_PKG_BUILD_DEPENDS="aspnetcore-targeting-pack-8.0, dotnet-targeting-pack-8.0"
+TERMUX_PKG_DEPENDS="aspnetcore-runtime-8.0, dotnet-host, dotnet-runtime-8.0"
+TERMUX_PKG_BUILD_IN_SRC=true
+TERMUX_PKG_SERVICE_SCRIPT=(
+	"jackett"
+	"${TERMUX_PREFIX}/bin/dotnet ${TERMUX_PREFIX}/lib/jackett/jackett.dll --DataFolder ${TERMUX_ANDROID_HOME}/.config/jackett"
+)
+TERMUX_PKG_BLACKLISTED_ARCHES="arm"
+
+termux_step_pre_configure() {
+	export DOTNET_CLI_TELEMETRY_OPTOUT=1
+	termux_setup_dotnet
+}
+
+termux_step_make() {
+	dotnet publish src/Jackett.Server --framework "net8.0" --no-self-contained --runtime $DOTNET_TARGET_NAME --configuration Release --output build/ /p:AssemblyVersion=${TERMUX_PKG_VERSION} /p:FileVersion=${TERMUX_PKG_VERSION} /p:InformationalVersion=${TERMUX_PKG_VERSION} /p:Version=${TERMUX_PKG_VERSION}
```

Comment:
Consider breaking this line up and quoting the variables.
```suggestion
	dotnet publish src/Jackett.Server \
	--framework "net8.0" \
	--no-self-contained \
	--runtime "$DOTNET_TARGET_NAME" \
	--configuration Release \
	--output build/ \
	/p:AssemblyVersion="${TERMUX_PKG_VERSION}" \
	/p:FileVersion="${TERMUX_PKG_VERSION}" \
	/p:InformationalVersion="${TERMUX_PKG_VERSION}" \
	/p:Version="${TERMUX_PKG_VERSION}"
```